### ISSUE TITLE
A tutorial page can list multiple pages of the same content 

### ISSUE THREAD

**Issue by 130s** [ros-infrastructure/roswiki#245]
E.g. from http://wiki.ros.org/roswtf/Tutorials

![screenshot from 2018-04-14 22-26-29](https://user-images.githubusercontent.com/1840401/38775271-f520c5cc-4032-11e8-8886-a4522fa46e0d.png)
Looks like the search macro captures pages translated to other languages.
```
<<FullSearchWithDescriptionsCS(title:"Tutorials/Getting started with roswtf")>>
```

**Comment by tfoote** [comment]
Yes, that's a known issue with FullSearch. The FullSearch* macros are also very hard on the wiki server (requiring opening every single wiki page on the server side). As such all instances of FullSearch macros should be removed. 

 All tutorial listings should be switched to the [TutorialChain Macro](http://wiki.ros.org/WikiMacros#TutorialChain)
